proxy/boot/manager-host.js

```diff
@@ -30,11 +30,21 @@ module.exports = function setupHooks(server) {
     next();
   });
 
-  ManagerHost.startPolling = function() {
-    ManagerHost.sync();
-    setInterval(function() {
-      ManagerHost.sync();
-    }, ManagerHost.settings.interval || 1000);
+  ManagerHost.startPolling = function(cb) {
+    var timer;
+    ManagerHost.sync(function(firstErr) {
+      if(firstErr) {
+        return cb ? cb(err) : null;
+      }
+      timer = setInterval(function() {
```

Comment:
I mean in the case where the app that is using strong-mesh-client is finished. There's no stopPolling, so the event loop will remain alive unless someone kills the process or the app calls `process.exit()`. This kind of problem comes up all the time with CLI apps and in tests.